projboard/templates/projboard/searchArticle/search_article.html

```diff
@@ -0,0 +1,79 @@
+{% extends "base.html" %}
+{% load static %}
+{% block content %}
+<head>
+    <link rel="stylesheet" type="text/css" href="{% static 'css/search_article.css' %}">
```

Comment:
Consider moving this line to base.html
because the design of the headers is not the same.

Reply:
In my opinion, you should move **only the common** styles to the all pages.
So if you think that you have some styles that common to all the website.. do it.

Reply:
Added common styles